/virtual_serial_driver:
  ros__parameters:
    bullet_speed: 26.5
    enemy_color: 82
    pitch_joint: 0.0
    yaw_joint: 0.0